test/src/specs/hardfork/v2021/cell_deps.rs

```diff
@@ -19,14 +22,11 @@ use std::fmt;
 const GENESIS_EPOCH_LENGTH: u64 = 10;
 const CKB2021_START_EPOCH: u64 = 10;
 
-const TEST_CASES_COUNT: usize = (8 + 4) * 3;
-const CELL_DEPS_COUNT: usize = 2 + 3 + 2;
-const INITIAL_INPUTS_COUNT: usize = 2 + CELL_DEPS_COUNT + TEST_CASES_COUNT;
+const TEST_CASES_COUNT: usize = 1;
+const CELL_DEPS_COST_COUNT: usize = (1 + 2 + 2 + 1) + (4 + 5 + 4);
```

Comment:
What do these magic numbers mean? Please add some comments for other maintainers.

Reply:
I add a comment for these numbers.

Reply:
```suggestion
// Count `new_with_id`, `create_tx_as_code_celldep`, and `create_depgroup_celldep` in `CheckCellDepsTestRunner::create_celldep_set()`. Each spends one input.
const CELL_DEPS_COST_COUNT: usize = 19;
```